packages/terra-date-picker/src/react-datepicker/stylesheets/react_datepicker.module.scss

```
@@ -430,7 +430,13 @@
       border: 1.8 * $datepicker-navigation-size solid transparent;
     }
   }
-
+  @media (max-width: 320px), (max-height: 256px) {
+    .react-datepicker-portal {
+      align-items: flex-start;
+      overflow: auto;
+    }
+  }
```

add a line break and comment for consistency 

```suggestion

  // Allows calendar to scroll on smaller viewports
  @media (max-width: 320px), (max-height: 256px) {
    .react-datepicker-portal {
      align-items: flex-start;
      overflow: auto;
    }
  }
```

---

Done https://github.com/cerner/terra-framework/commit/acd9d0dfd0f8928b39ceb4dedbb003c77806849d